integration/shared/isolated/enable_service_access_command_test.go

```diff
@@ -190,6 +194,65 @@ var _ = Describe("enable service access command", func() {
 				})
 			})
 
+			When("two services with the same name are registered", func() {
```

Comment:
This `When` block should only be run when the targeted API version supports registering multiple services with the same name.